install_config/master_node_configuration.adoc

```diff
@@ -1784,11 +1777,17 @@ master-logs api api 2> file
 [[master-node-config-logging-levels]]
 === Configuring Logging Levels
 
-You can control which INFO messages are logged by setting the `DEBUG_LOGLEVEL` option in the in xref:../admin_guide/manage_nodes.adoc#modifying-nodes[node configuration files] or the */etc/origin/master/master.env* file. Configuring the logs to collect all messages can lead to large logs that are difficult to interpret and can take up excessive space. Only collect all messages when you need to debug your cluster.
+You can control which INFO messages are logged by setting the `DEBUG_LOGLEVEL`
+option in the in xref:../admin_guide/manage_nodes.adoc#modifying-nodes[node configuration files]
```

Comment:
Strange wording `in the in`

Reply:
I didn't see `admin_guide/manage_nodes.adoc#modifying-nodes` there mentions log level, is this link good?

Reply:
Fixed typo and removed link.